docs/source/api_ref_models.rst

```diff
@@ -133,6 +133,29 @@ To download the CodeLlama-7B model:
     code_llama2.lora_code_llama2_70b
     code_llama2.qlora_code_llama2_70b
 
+qwen-2
+------
+
+Models from the `Qwen2 small model family <https://huggingface.co/collections/Qwen/qwen2-6659360b33528ced941e557f>`_.
+
+To download the Qwen2 1.5B model, for example:
+
+.. code-block:: bash
+
+    tune download Qwen/Qwen2-0.5B-Instruct --output-dir /tmp/Qwen2-0.5B-Instruct --ignore-patterns None
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
```

Comment:
we also usually expose the component builders, but let's chat about what the best way to doc this is in the model page PR, I think we agree this doesn't look very clear to the uninitiated
```suggestion
    qwen2.qwen2
    qwen2.lora_qwen2
```